opal/util/event.h

```diff
@@ -45,8 +45,6 @@ BEGIN_C_DECLS
 
 typedef event_callback_fn opal_event_cbfunc_t;
 
-BEGIN_C_DECLS
```

Comment:
Please help me understand why this is removed but the matching `END_C_DECLS` is not.

Reply:
Well, because there were two `BEGIN_C_DECLS` and only one `END_C_DECLS`.

Reply:
Thanks I missed that.